pkg/apis/scheduling/install/install.go

```diff
@@ -37,12 +38,13 @@ func Install(groupFactoryRegistry announced.APIGroupFactoryRegistry, registry *r
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:                  scheduling.GroupName,
-			VersionPreferenceOrder:     []string{v1alpha1.SchemeGroupVersion.Version},
+			VersionPreferenceOrder:     []string{v1beta1.SchemeGroupVersion.Version, v1alpha1.SchemeGroupVersion.Version},
 			RootScopedKinds:            sets.NewString("PriorityClass"),
```

Comment:
Here it says that rootscopekind is PriorityClass? So wondering why to keep both v1alpha1 and v1beta1 if its only for priorityclass?

Reply:
I am not a 100% sure. I believe it is because of backward compatibility. I have looked at existing API which have graduated to beta and all of them have this pattern of beta and alpha.

Reply:
In my understanding, there is no guarantee provided by Kubernetes of backward compatibility for alpha apis. See the alpha api section: https://kubernetes.io/docs/concepts/overview/kubernetes-api/